src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -339,6 +349,9 @@ public String getFieldGetFunctionName(TypeRef type, Name identifier) {
 
   /** The function name to get a field having the given type and name. */
   public String getFieldGetFunctionName(FieldModel type, Name identifier) {
+    if (type.getApiSource().equals(ApiSource.DISCOVERY)) {
```

Comment:
I assume we'll be able to move this source-specific logic out of `SurfaceNamer` eventually? (Maybe add a TODO if so)